packages/ui/src/lib/NativeBase/Navbar.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  Image,
+  Box,
+  HamburgerIcon,
+  useMediaQuery,
+  HStack,
+  Button,
+  Link,
+  Square,
+} from "native-base";
+import PropTypes, { InferProps } from "prop-types";
+import React, { useReducer } from "react";
+
+const menuItemPropTypes = {
+  name: PropTypes.string.isRequired,
+  url: PropTypes.string.isRequired,
+};
+
+type MenuItemProps = InferProps<typeof menuItemPropTypes>;
+
+function MenuItem({ name, url }: MenuItemProps) {
+  return (
+    <Box alignItems="flex-start">
+      <Link
+        href={url}
+        px="16px"
+        py="10px"
+        fontSize="16px"
+        isUnderlined={false}
+        _text={{ color: "#FFFFFF" }}
+        _hover={{ _text: { color: "#00C2FF" } }}
+      >
+        {name}
+      </Link>
+    </Box>
+  );
+}
+
+MenuItem.propTypes = menuItemPropTypes;
+
+const navbarPropTypes = {
+  items: PropTypes.arrayOf(PropTypes.shape(menuItemPropTypes)).isRequired,
+  logo: PropTypes.shape({
+    source: PropTypes.string.isRequired,
+    url: PropTypes.string.isRequired,
+  }).isRequired,
+};
+
+type NavbarProps = InferProps<typeof navbarPropTypes>;
+
+export default function Navbar({ items, logo }: NavbarProps) {
+  const [isDesktop] = useMediaQuery({ minWidth: 768 });
```

Comment:
Instead of mediaQuery, let's solve this problem with styled system, see https://docs.nativebase.io/responsive